job-search-web-scraping.py

```diff
@@ -19,16 +19,26 @@ def indeed_job_search():
     browser.implicitly_wait(5) 
 
     search_results = browser.find_elements_by_xpath('//h2/a')
-
+    search_descriptions=browser.find_elements_by_xpath('//div[@class="summary"]')
```

Comment:
Format code added to match surrounding code. Add whitespace around operators. For example, edit "search_descriptions=browser" to be "search_descriptions = browser"